db/system_keyspace.cc

```diff
@@ -2089,7 +2102,7 @@ std::vector<schema_ptr> system_keyspace::all_tables(const db::config& cfg) {
                     v3::cdc_local(),
     });
 
-    r.insert(r.end(), {raft(), raft_snapshots(), raft_snapshot_config(), group0_history(), discovery()});
+    r.insert(r.end(), {raft(), raft_snapshots(), raft_snapshot_config(), group0_history(), discovery(), service_levels_v2()});
```

Comment:
I think you need extra magic to get the table to use the schema commitlog rather than the standard commitlog.